WowPacketParser/Store/Objects/Creature.cs

```diff
@@ -16,33 +16,27 @@ public sealed class Creature : IDataModel
         [DBFieldName("map")]
         public uint? Map;
 
-        [DBFieldName("zoneId")]
```

Comment:
Don't rename any of these, it must insert into TC database

Reply:
Minor oversight from merging his changes.

Reply:
My idea was for the WPP database to be its own thing, so I can export as much useful data to sql tables as possible, and to remove all placeholder values that don't come directly from sniffs. Since I don't really work with TC, there was no benefit to keeping stuff in sync with TDB, and I went ahead and renamed a bunch of fields to the lowercase underscore style I use in my own project. This isn't something I intended to submit to the main repository.